site2/docs/getting-started-standalone.md

```diff
@@ -16,7 +16,7 @@ If you're looking to run a full production Pulsar installation, see the [Deployi
 
 - JRE (64-bit). Different Pulsar versions rely on different JRE versions. For how to choose the JRE version, see [Pulsar Runtime Java Version Recommendation](https://github.com/apache/pulsar/blob/master/README.md#pulsar-runtime-java-version-recommendation).
 
-## Step 1. Download Pulsar distribution
+## Download Pulsar distribution
```

Comment:
Why remove "Step"? Comments see https://github.com/apache/pulsar/pull/17475#discussion_r963198907

This is what we look like:
<img width="1837" alt="image" src="https://user-images.githubusercontent.com/50226895/189014101-2249f20e-7dc5-463c-9a8e-e2a7ee5fb434.png">

Other examples:
https://kafka.apache.org/quickstart
<img width="975" alt="image" src="https://user-images.githubusercontent.com/50226895/189014195-44e160ff-9884-434e-a754-67e540fea947.png">

Reply:
> Why remove "Step"? Comments see [#17475 (comment)](https://github.com/apache/pulsar/pull/17475#discussion_r963198907)
> 
> This is what we look like: <img alt="image" width="1837" src="https://user-images.githubusercontent.com/50226895/189014101-2249f20e-7dc5-463c-9a8e-e2a7ee5fb434.png">
> 
> Other examples: https://kafka.apache.org/quickstart <img alt="image" width="975" src="https://user-images.githubusercontent.com/50226895/189014195-44e160ff-9884-434e-a754-67e540fea947.png">

My two cents: Use the h2 as the procedure title, with numbered steps in regular paragraph text beneath. If there is only one step for each section, as is the case in your rewrite, it is not numbered.

Reply:
Sorry, I do not get your point, can you please explain a bit more? Thanks!

Reply:
Like this example.

https://docs.google.com/document/d/1rKmmJ5oj-fiK2uwYScqU7aiOys7q-fTjjJeLPCZcxfI/edit#

Reply:
I understand your proposed solution. I mean why remove "Step"? Is it clearer to show "Step xx" at the beginning?

Reply:
If we use "Step", can you revert this change?